tests/vm_test.go

```diff
@@ -1188,6 +1189,80 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(strings.HasPrefix(stdErr, "Error from server (NotFound): virtualmachines.kubevirt.io")).To(BeTrue(), "should fail when deleting non existent VM")
 		})
 
+		Context("as ordinary OCP user", func() {
+
+			const (
+				testUser    = "testuser"
+				testPass    = "1234567890"
```

Comment:
I am sure I am missing something, but: who is responsible for creating the user? I can't find it mentioned in this PR, in the repo, nor in kubevirtci repo. Someone must create it since we are using it here.

Reply:
Hi @booxter, the user is created by first login on line 1204.
Openshift in our CI is configured for allow all, that means  allow any user with any login to enter the cluster.
If the user is not already present, it is created with default rules. If it is in default namespace, the user does not have any special right and cannot do anything.
Once he creates his projects, he inherits the project defaults, enabling him to operated pods and kubevirt.

This was the aim of this test.